code/game/objects/items/fluff.dm

```diff
@@ -0,0 +1,296 @@
+/*		FLUFFY FUN
```

Comment:
tennis balls and frisbees should be moved to the `code/modules/projectiles/ammunition/caseless` folder, and the projectiles to the `code/modules/projectiles/projectile/reusable` one.